tests/migrations/test_base.py

```diff
@@ -168,6 +170,13 @@ def assertFKExists(self, table, columns, to, value=True, using="default"):
     def assertFKNotExists(self, table, columns, to):
         return self.assertFKExists(table, columns, to, False)
 
+    def assertFormatterFailureCaught(
+        self, command, *args, module="migrations.test_migrations", **kwargs
```

Comment:
`command` is always going to be `call_command`, so maybe we don't need to parameterize it? It could be confused for the strings. "migrate", "makemigrations", etc. which are commands.

Reply:
Good point! Wanna push an update? (please fetch my forced update first)